spec/factories/course_option.rb

```diff
@@ -1,6 +1,8 @@
 FactoryBot.define do
   factory :course_option do
+    initialize_with { CourseOption.find_or_initialize_by(course: course, site: site) }
```

Comment:
Will this do the right thing if we want a course option with a different study mode?

Reply:
ok great :+1: